util/sfc_climo_gen/run.hera.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#------------------------------------------------------------
+# Run the sfc_climo_gen program stand-alone on Hera using
+# pre-exiting 'grid' and 'orography files.
+#
+# Set the configuration variables in sfc_gen.sh. Then
+# run this script as follows: 'sbatch $script'
+#------------------------------------------------------------
+
```

Comment:
The pre-existing 'grid' and 'orography' files need to be staged in FIX_FV3=${BASE_DIR}/fix/orog/C${res}, right?  Can that info be added to the comment block above, including the names of the files?

Reply:
Added more comments at 5baa903.